### DISCUSSION TITLE
Recovery - Is it possible ?

### DISCUSSION THREAD

**Discussion by lahatch** [cloudnative-pg/cloudnative-pg#6329] [Q&A] [answered]
Hi all, I looked at the forum and I didn't find the answer to my question but maybe I didn't see it, sorry for that -|

Using clounativepg/cluster helm chart, I deployed a postgresql cluster, hosted into an AWS EKS-A (under AWS account-A), with a scheduledbackup configured using an S3 bucket. This works fine, all the base files and wals are present in the s3 bucket.

I would like to create a recovery cluster, hosted into an AWS EKS-B, under AWS account-B with no communication, no vpc peering between EKS-A and EKS-B.

I didn't find how to do that.
Is it possible because when reading the documentation, it's seems that "Recovers a CNPG cluster from a CNPG backup (PITR supported) Needs to be on the same cluster in the same namespace"

Any help will be appreciated.
Thanks
Rachel

**Comment by alexbarcelo** [comment, marked as answer]
Have you read the "Distributed Topology" section from [Replica clusters](https://cloudnative-pg.io/documentation/current/replica_cluster/)?

Creating a recovery cluster is possible, and the general outline described there seem to match your requirements. Note that you need an S3 bucket for the AWS EKS-A **and** a bucket for the AWS EKS-B. And you will also need to configure the accordingly the `externalClusters` for both with both. And make use of the `recovery` section (see "Bootstrap the Replica Cluster in that documentation I am linking) for starting the replica instance.

I mention Distributed Topology but if you will never need to transfer primary-ness from one place to another, maybe the simpler (?) Standalone Replica Clusters work... both are described in the same documentation URL.

**Reply by lahatch** [reply]
yes, thank you alexbarcelo, indeed it seems to match my requirements, I missed it! I will try :)
